package.json

```diff
@@ -54,6 +54,7 @@
     "mocha-junit-reporter": "^2.0.0",
     "mocha-multi-reporters": "^1.5.1",
     "prettier": "2.4.1",
+    "punycode": "^1.4.1",
```

Comment:
The `url` polyfill package declares punycode as a dependency

https://github.com/defunctzombie/node-url/blob/5480ec001102457ac00c3c7878facdec39b536c8/package.json#L6-L9

but then does `require("punycode")` which uses the node polyfill (as opposed to `require("punycode/")`). I needed to declare this version to get the integration tests to pass....